### PR TITLE
feat(triggers): creates landing page

### PR DESCRIPTION
**Pull request by jurajpiar**
Creates initial landing page for triggers aka notifications.

Resolves [RMKT-515](https://rsklabs.atlassian.net/browse/RMKT-515)

### CODE REVIEW

File: src/components/pages/triggers/TriggersRoutes.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  useEffect(() => {
+    // TODO: wrap this code block to be excecuted only on debugging
+    const unlisten = history.listen((location, action) => {
+      logger.debug('TriggersRoutes -> location', location)
+      logger.debug('TriggersRoutes -> action', action)
+    })
+    return (): void => {
+      unlisten()
+    }
+  }, [history])
```
Comment (farinalvaro) [inline src/components/pages/triggers/TriggersRoutes.tsx:28]: may be we can remove this and add it if needed? not sure about propagating this, or may be move to approutes?

File: src/components/pages/triggers/TriggersLandingPage.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
+import { Typography } from '@material-ui/core'
+import Icon, { Icons } from 'components/atoms/Icon'
+import SideImageTemplate from 'components/templates/SideImageTemplate'
+import React, { FC } from 'react'
+
+export type TriggersLandingPageProps = {
+
+}
```
Comment (farinalvaro) [inline src/components/pages/triggers/TriggersLandingPage.tsx:8]: Remove empty type

File: src/components/pages/triggers/TriggersLandingPage.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+const TriggersLandingPage: FC<TriggersLandingPageProps> = () => {
+  const icon = <Icon src={Icons.TRIGGERS} />
+  const sideText = (
+    <>
+      <Typography variant="body1">
+        <b>RIF Notification services</b>
```
Comment (farinalvaro) [inline src/components/pages/triggers/TriggersLandingPage.tsx:15]: Prefer the use of `Box` component. Example: https://github.com/rsksmart/rif-marketplace-ui/blob/master/src/components/templates/TabsTemplate.tsx#L34

**Review by itofarina — COMMENTED** [review]
Nice! Just left a couple of suggestions

File: src/components/pages/triggers/TriggersRoutes.tsx (hunk 1 above)
Comment (jurajpiar) [reply src/components/pages/triggers/TriggersRoutes.tsx:28]: No, this was a conscious decision. we'll refactor all of the Routers in one PR. So it doesn't matter and in the mean time it at least gives us the same info as the rest of the routers.

**Review by itofarina — APPROVED** [review]

File: src/components/organisms/ServiceCategories.tsx [hunk 4]
```diff
@@ -27,17 +27,23 @@ const ServiceCategories: FC<ServiceCategoriesProps> = () => {
...
     {
-      to: ROUTES.STORAGE.BUY.BASE,
+      to: ROUTES.STORAGE.BASE,
       text: 'Storage',
       iconProps: { src: Icons.STORAGE, alt: 'Storage icon' },
       description: 'Offer/Rent Decentralized Storage through the RIF Marketplace!',
     },
+    {
+      to: ROUTES.TRIGGERS.BASE,
+      text: 'Notificaions',
```
Comment (julianmrodri) [inline src/components/organisms/ServiceCategories.tsx:43]:
```suggestion
      text: 'Notifications',
```

File: src/components/organisms/Header.tsx [hunk 5]
```diff
@@ -59,15 +60,27 @@ const Headers: FC = () => {
...
       title: 'Storage',
       to: ROUTES.STORAGE.BASE,
-      isActive: (_, { pathname }) => pathname.includes(ROUTES.STORAGE.BASE),
+      isActive: (_, {
+        pathname,
+      }): boolean => pathname.includes(ROUTES.STORAGE.BASE),
       icon: <StorageIcon />,
     },
+    {
+      title: 'Notificaions',
```
Comment (julianmrodri) [inline src/components/organisms/Header.tsx:77]:
```suggestion
      title: 'Notifications',
```

File: src/components/organisms/ServiceCategories.tsx [hunk 6]
```diff
@@ -27,17 +27,23 @@ const ServiceCategories: FC<ServiceCategoriesProps> = () => {
...
+      iconProps: { src: Icons.TRIGGERS, alt: 'Notificaions icon' },
```
Comment (julianmrodri) [inline src/components/organisms/ServiceCategories.tsx:44]:
```suggestion
      iconProps: { src: Icons.TRIGGERS, alt: 'Notifications icon' },
```

File: src/routes.ts [hunk 7]
```diff
@@ -17,6 +17,9 @@ const STORAGE_MYOFFERS_CANCEL = `${STORAGE_MYOFFERS}/cancel`
 const STORAGE_MYPURCHASES = `${STORAGE}/mypurchases`
 const STORAGE_RENEW_AGREEMENT = `${STORAGE_MYPURCHASES}/renew`
 
+const TRIGGERS = '/notificaions'
```
Comment (julianmrodri) [inline src/routes.ts:20]: We should fix here to `/notifications`

**Review by julianmrodri — COMMENTED** [review]
Left some comments.

File: src/components/pages/triggers/TriggersRoutes.tsx (hunk 1 above)
Comment (jurajpiar) [reply src/components/pages/triggers/TriggersRoutes.tsx:28]: https://github.com/rsksmart/rif-marketplace-ui/issues/696

**Pull request merged**